ludwig/contribs/whylogs/__init__.py

```diff
@@ -0,0 +1,20 @@
+from ludwig.callbacks import Callback
+from whylogs import get_or_create_session
+
+
+class WhyLogsCallback(Callback):
+    def __init__(self, path_to_config=None):
+        self.session = get_or_create_session(path_to_config)
+        self.logger = self.session.logger()
+
+    def on_build_metadata_start(self, df):
+        self.logger.log({'metadata_start': df})
+
+    def on_build_metadata_end(self, df):
+        self.logger.log({'metadata_end': df})
```

Comment:
We probably only need to log once during `on_build_metadata_start`. The other callbacks we can just leave for others who may be interested in overriding them.

Reply:
Makes sense. This in combo with change from your previous comment makes sense.